### ISSUE TITLE
Crashing French translations

### ISSUE THREAD

**Issue by Startouf** [kslazarev/numbers_and_words#107]
I try to translate some figures into French, but I run into a 

uninitialized constant NumbersAndWords::Strategies::FiguresConverter::Decorators::Fr error

Is there an option somewhere to switch back to english translations if a problem like this appears ?

**Comment by dblock** [comment]
It looks like this is a legit bug. It would be great if you could build a repro spec for this? Thanks.

**Comment by Startouf** [comment]
It's only about float numbers conversion, and apparently :en language fails too. This might be related to my gems, but I have different error for :en `I18n::InvalidPluralizationData: translation data {:one=>"hundredth", :other=>"hundredths"} can not be used with :count => 42`
 error for :en language

Below is a test for `:fr` translations only, but maybe you could do something like `[:fr, :de, :en, :....].each do` to test all languages (in case the bug only appears for some specific untranslated language). I have never written tests so far, so I hope this is good.

```ruby
test "it should translate integer and float numbers or fallback to a known language traslation" do
  assert_block("It works with integers") do # Note this one already passes (for :fr at least)
    # Ideally add a loop here to test for all languages of the I18n
    I18n.with_locale(:fr) do
      assert_block("Integer translation for :fr") do
        42.to_words == "quarante-deux" or 42.to_words == "forty-two" # Replace by default translation
      end
    end
  end
  assert_block("It works with floats") do # This is the one that doesn't
    # Ideally add a loop here to test for all languages of the I18n
    I18n.with_locale(:fr) do
      assert_block("Float translation for :fr") do
        (42.42).to_words == "quarante-deux virgule quarante-deux" or (42.42).to_words == "forty-two coma forty-two" # Replace by default translation
      end
    end
  end
end
```

**Comment by sebfie** [comment]
Any solution for that?

**Comment by bsylvain** [comment]
I have the same problem, is there any solution?

**Comment by Startouf** [comment]
I had fixed the problem on my fork of the repo. It was quite a long time ago, and I'm not even sure it's still compatible with the current master.
But you might want to try [https://github.com/Startouf/numbers_and_words] to see if it fixes your problem

**Comment by dblock** [comment]
There's a PR in https://github.com/kslazarev/numbers_and_words/pull/111, but it needs tests. Someone wants to finish it up?

**Comment by jlduran** [comment]
Closed in #144
